<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapExport</class>
 <widget class="QDialog" name="MapExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Map</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>position attribute</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="4">
    <widget class="QComboBox" name="comboBoxPositionSelection">
     <item>
      <property name="text">
       <string>-select attribute-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="pushButtonOutputSelection">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxBinary">
     <property name="text">
      <string>binary</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxCompress">
     <property name="text">
      <string>compress</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="4">
    <widget class="QComboBox" name="comboBoxMapSelection">
     <item>
      <property name="text">
       <string>-select map-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>other vertex attributes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>other cell attributes</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="4">
    <widget class="QListWidget" name="listWidgetVertexAttributes"/>
   </item>
   <item row="4" column="3" colspan="4">
    <widget class="QListWidget" name="listWidgetCellAttributes"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>output</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QLineEdit" name="lineEditOutput">
     <property name="text">
      <string>-select output-</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
